depthai_sdk/docs/source/blob_nnet_manager.rst

```diff
@@ -0,0 +1,76 @@
+======================
+Blob and Nnet managers
+======================
+
+Blob manager
+------------
+
+``Blob manager`` is a class that is made to help you with downloading neural networks as MyriadX blobs.
+
+Getting started
+^^^^^^^^^^^^^^^
+
+``Blob manager`` is very easy and straight forward to use. We declare it and pass which project we want to use as it's argument.
+
+.. literalinclude:: ./examples/code_fractions/blob.py
+   :language: python
+   :linenos:
+
+After that, the ``blob`` is stored in our variable and we can then pass it to our ``Nnet manager``, as we will se bellow.
+Of course the ``face-detection-retail-0004`` is not the only project that we can use. All other projects that are supported are:
+
+.. include::  blob_projects.rst
+
+Nnet manager
+------------
+
+``Blob manager`` is a class that is made to help you with setting up neural networks.
+
+Getting started
+^^^^^^^^^^^^^^^
+
+As the ``Blob manager`` and ``Nnet manager`` work together, we will need both to use our neural network. We will also need to use the ``Pipeline manager`` and the ``Preview manager`` to make everything work together.
+
+``Pipeline manager`` will define our stream sources from our OAK camera,
+
+``Preview manager`` will show us our frames (to see what is happening),
+
+``Blob manager`` will download and store our blob file,
+
+and ``Nnet manager`` will start and run our neural network.
+
+Face detection
+^^^^^^^^^^^^^^
+
+.. literalinclude:: ./examples/face_detection_color.py
+   :language: python
+   :linenos:
+
+In this above example we will use all above mentioned classes and run the face detection project.
+First we define the pipeline and initialize the streams. After that we load in our blob (``face-detection-retail-0004``) and send it in to our ``Nnet manager``.
+Every project has its own ``inputSize`` and ``familyName``. To see more about the different requirements check the ``depthai/resources/nn`` folder. Every blob has its requirements in it's json file.
```

Comment:
`depthai/respources/nn` are specific to depthai demo, it's also not needed when dealing with mobilenet-ssd-based networks (like the one you used), all parameters can be supplied on init.
Maybe nitpicking, but these json files store decoding config, not actual requirements